artifactory_cleanup/artifactorycleanup.py

```diff
@@ -25,20 +26,24 @@ def __init__(
         today: date,
         ignore_not_found: bool,
         worker_count: int,
+        output_format: str,
+        output_artifacts: bool,
```

Comment:
Cleanup doesn't have to know about output, it works and returns in memory objects
```suggestion
```